Gemfile.lock

```diff
@@ -5,6 +5,7 @@ GEM
       public_suffix (>= 2.0.2, < 4.0)
     colorator (1.1.0)
     ffi (1.10.0)
+    ffi (1.10.0-x64-mingw32)
```

Comment:
@nkprince007 to check this part 😬 
I have no idea

Reply:
Looks to me like some windows specific dependency. Let it stay for now.